Microsoft.Toolkit.Uwp.UI/Shadows/AttachedShadowBase.cs

```diff
@@ -174,9 +174,12 @@ public AttachedShadowElementContext GetElementContext(FrameworkElement element)
         /// <inheritdoc/>
         public IEnumerable<AttachedShadowElementContext> EnumerateElementContexts()
         {
-            foreach (var kvp in ShadowElementContextTable)
+            if (ShadowElementContextTable != null)
```

Comment:
I know this prevents the crash, but I think there's still the root cause of why this is null when it is here in the particular scenario with the call stack. As I don't think we ever expect this to be null? (@Ryken100 thoughts?)

Reply:
`ShadowElementContextTable` isn't initialized until the shadow is attached to at least one element. `AttachedDropShadow.OnCastToPropertyChanged()` calls `EnumerateElementContexts()`, so perhaps that's where the crash happens?

Since `EnumerateElementContexts()` is a public method and since `ShadowElementContextTable` is lazy initialized, it may be difficult for callers to know when `ShadowElementContextTable` is initialized, so I think the null check is the right choice here.

Reply:
Yes, The crash happens on EnumerateElementContexts(), For more info on the issue #4750